src/main/java/org/candlepin/subscriptions/tally/MetricUsageCollector.java

```diff
@@ -99,17 +103,39 @@ public ServiceLevel getDefaultSla() {
         }
     }
 
+    @Autowired
     public MetricUsageCollector(AccountRepository accountRepository,
-        EventController eventController) {
+        EventController eventController, ApplicationProperties applicationProperties) {
 
         this.accountRepository = accountRepository;
         this.eventController = eventController;
         this.productConfig = new ProductConfig();
+        this.applicationProperties = applicationProperties;
+
+    }
+
+    protected OffsetDateTime adjustTimeForLatency(OffsetDateTime dateTime, Duration adjustmentAmount) {
+
+        return dateTime.toZonedDateTime().minus(adjustmentAmount).toOffsetDateTime();
+
```

Comment:
Thought about it some, and I think we should just do some validation of the ranges passed to these methods in hit tasks, and save the adjustment exclusively for ENT-3542. I find it a little confusing to apply it to anything _but_ the scheduler.

Reply:
I agree.  Will adjust this on the ENT-3542 branch (#299)